jetty-core/jetty-ee/jetty-ee-webapp/pom.xml

```diff
@@ -50,7 +50,7 @@
         <configuration>
           <argLine>@{argLine} ${jetty.surefire.argLine} --add-reads org.eclipse.jetty.ee.webapp=org.eclipse.jetty.logging</argLine>
           <excludes>
-            <!-- This class modifies internal JVM statics, and causes problems in parallel testing -->
+            <!-- This class modifies internal JVM statistics, and causes problems in parallel testing -->
```

Comment:
I don't think this was a typo, looks like the `WebAppClassLoaderUrlStreamTest` is trying to change a static field inside of the `java.net.URL` class.